### ISSUE TITLE
Loss some features when using sklearn2pmml

### ISSUE THREAD

**Issue by STHSF** [jpmml/sklearn2pmml#222]
I using PMMLPipeline() to train lightgbm model, and save PMML model with sklearn2pmml. But I find some features not in PMML file

**Comment by vruusmann** [comment]
The PMML representation does not store REDUNDANT features.

For example, if your training dataset has 100 columns but your pipeline contains a `SelectKBest(k = 5)` step, then the PMML file will contain exactly five features (not 100 features).

There is no point in converting, storing and computing features that do not contribute towards the prediction. Your model is making correct predictions now and at all other times.

TLDR: This is a feature, not a bug.

**Comment by STHSF** [comment]
I know, but i didn't use `SelectKBest ()` or other feature selection methods.

**Comment by vruusmann** [comment]
Was using `SelectKBest` as an example. The same goes for modeling algorithms that perform feature selection naturally - all sorts of regularized regressions, and decision tree models.

If a feature is not needed for anything, then it is deleted on purpose.

**Comment by STHSF** [comment]
Thank you
